controller/src/test/java/io/pravega/controller/server/PravegaTablesControllerServiceMainTest.java

```diff
@@ -104,46 +92,22 @@ public void testZKSessionExpiry() throws Exception {
                 });
 
         controllerServiceMain.startAsync();
-
-        try {
-            controllerServiceMain.awaitRunning();
-        } catch (IllegalStateException e) {
-            log.error("Failed waiting for controllerServiceMain to get ready", e);
-            Assert.fail("Failed waiting for controllerServiceMain to get ready");
-        }
+        controllerServiceMain.awaitRunning();
 
         MockControllerServiceStarter controllerServiceStarter = (MockControllerServiceStarter) controllerServiceMain.awaitServiceStarting();
-        try {
-            controllerServiceStarter.awaitRunning();
-        } catch (IllegalStateException e) {
-            log.error("Failed waiting for controllerServiceStarter to get ready", e);
-            Assert.fail("Failed waiting for controllerServiceStarter to get ready");
-            return;
-        }
+        controllerServiceStarter.awaitRunning();
 
         assertEquals(1, clientQueue.size());
         CuratorFramework curatorClient = (CuratorFramework) clientQueue.poll().getClient();
         // Simulate ZK session timeout
         // we will submit zk session expiration and 
-        CompletableFuture.runAsync(() -> {
```

Comment:
this was originally injecting the session expiration and making the zk client call asynchronously. 

although looking at the code, it doesnt make difference.